docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '2.1'
+
+volumes:
+  db:
+
+services:
+
+  db:
+    image: mysql
+    env_file: .env
+    environment:
+      - MYSQL_ROOT_PASSWORD=${DB_PASSWORD}
+      - MYSQL_DATABASE=roadmap
+    healthcheck:
+      test: mysqladmin -uroot -p${DB_PASSWORD} ping && [ -e /var/lib/mysql/roadmap ]
+      interval: 5s
+      timeout: 5s
+      retries: 100
+
+    volumes:
+      - db:/var/lib/mysql
+
+  web:
+    build: .
+    image: ualbertalib/dmp_roadmap
+    env_file: .env
```

Comment:
same as above, what's the purpose of env_file?

Reply:
this way all passwords are saved in one place, but sure, I will remove it.

Reply:
You didn't save all passwords in this file, you set them in environment variables here:
`environment:
     - MYSQL_ROOT_PASSWORD=${DB_PASSWORD}
    - MYSQL_DATABASE=roadmap`

and the password you set in env_file was a dummy. 

I'm fine if you prefer to use .env_file to keep all passwords in file. But if so, please use it properly.

Reply:
OK change .env file to keep ALL environment variables there, this makes more sense